### PR TITLE
Proofread the Service Worker documents and corrected the unnatural part

### PR DESCRIPTION
**Pull request by joshua1988**
@petele - Thanks for your work. Wondering if I update the contents, does my profile info get updated too? Can't see my pictures on the bottom of translated pages. :)

### CODE REVIEW

**Comment by petele** [comment]
@cwdoh - would you take a quick look and make sure it all makes sense?

**Comment by joshua1988** [comment]
@petele, thanks for asking him to review the documents. Do you think that profile picture doesn't get updated due to some errors in the contents?

File: src/content/ko/fundamentals/primers/service-worker/index.markdown [hunk 1]
```diff
@@ -26,5 +26,5 @@ translators:
 * [JavaScript Worker](http://www.html5rocks.com/en/tutorials/workers/basics/) 입니다,
   그렇기 때문에 DOM을 직접적으로 접근할 수 없습니다. 대신, 서비스워커는 페이지와 [postMessage](https://html.spec.whatwg.org/multipage/workers.html#dom-worker-postmessage) 인터페이스를 통해 커뮤니케이션이 가능하고, 필요하면 페이지들이 DOM을 조작할 수 있습니다.
 * 프로그래밍이 가능한 네트워크 프록시 입니다. 페이지의 네트워크 요청 처리를 제어할 수 있습니다.
-* 사용하지 않을 때는 종료되고, 다음번에 필요하면 재시작됩니다. 서비스워커의 `onfetch`와 `onmessage` 핸들러의 글로벌 스테이트에 의존할 수 없습니다. 만약 유지해야 하는 정보가 있고 재시작시 다시 사용해야 한다면, [IndexedDB API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)를 이용합니다.
+* 사용하지 않을 때는 종료되고, 다음번에 필요하면 재시작됩니다. 서비스워커의 `onfetch`와 `onmessage` 핸들러의 전역 상태에 의존할 수 없습니다. 만약 유지해야 하는 정보가 있고 재시작시 다시 사용해야 한다면, [IndexedDB API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)를 이용합니다.
```
Comment (cwdoh) [inline src/content/ko/fundamentals/primers/service-worker/index.markdown]: 좀 더 친절한 설명을 위해 `전역 상태에 의존하여 정보를 유지할 수 없습니다.`로 설명하면 어떨까 싶습니다.

File: src/content/ko/fundamentals/primers/service-worker/install.markdown [hunk 2]
```diff
@@ -6,17 +6,17 @@ translators:
   - captainpangyo
 ---
 
-페이지가 등록 절차를 시작하면 서비스워커 스크립트에서 `install` 이벤트를 처리하는 부분에 집중합니다.
+페이지가 등록 절차를 시작하면 `install` 이벤트를 처리하는 서비스워커 스크립트를 봅니다.
```
Comment (cwdoh) [inline src/content/ko/fundamentals/primers/service-worker/install.markdown]: 본다는 표현은 실행/연동의 의미로 조금 의역하는 것이 어떨지 :)

**Comment by cwdoh** [comment]
@petele - After quick proofreading, looks good to me.
Trivial translation issues will be discussed at offline fireside chat. :)
Thanks for translations, @joshua1988.

File: src/content/ko/fundamentals/primers/service-worker/install.markdown (hunk 2 above)
Comment (joshua1988) [inline src/content/ko/fundamentals/primers/service-worker/install.markdown]: 봅니다가 더 깔끔하겠네요! 감사합니다

File: src/content/ko/fundamentals/primers/service-worker/index.markdown (hunk 1 above)
Comment (joshua1988) [inline src/content/ko/fundamentals/primers/service-worker/index.markdown]: 제가 경험이 부족해서 아직은, 어느 걸 더 명확한 의미 전달을 위해 발음 그대로 써야 하고, 어떤 것은 의역을 해줘야 하는지 잘 모르겠네요 :) 코멘트 감사합니다!

**Comment by petele** [comment]
@joshua1988 - I'm not sure why your contributor image isn't showing up. Either the contributors.yaml file is corrupt (my first guess), or it can't find your image file. The image name should match the key your used, for example, my key is `petelepage` and there's a `petelepage.jpg` in the contributors image directory.

Feel free to take a look and see if you can figure it out, and if you don't have time, file a bug and assign to me.

**Comment by petele** [comment]
@cwdoh - huge thanks!

**Comment by joshua1988** [comment]
@petele - That was my thought too to check the image file name and contributors.yaml file. I just have double checked those two files and they don't seem to be wrong. As you mentioned, I will file a bug and assign it to you. Thanks for the kind explanation though! :)

**Pull request merged**